.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,130 @@
+name: Publish Release
+run-name: Publish Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release version (e.g. 1.2.3)'
```

Comment:
@planv this should probably be more informative

Reply:
Something like `Please type release version (e.g. 1.2.3)` ?

Reply:
@planv I would recommend checking out https://github.com/udx/udx-worker/blob/latest/.github/workflows/build.yml 

I used `gittools/actions/gitversion/execute@v0` for that purpose. I'm not sure if it's what you need, though.

Reply:
Will be reviewed for the next version, closing for now